docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -105,103 +105,103 @@ \section{Failure Mode and Effect Analysis}
   Email is duplicated &
   \begin{tabular}[c]{@{}l@{}}Compare the email \\ entered with the user \\ database records\\ to see if the email is in use\end{tabular} &
   \begin{tabular}[c]{@{}l@{}}Notify the user that the email \\ is associated with another\\  account\\ \\ Prompt them to give another \\ email or sign in with the one \\ they entered\end{tabular} &
-   &
+   SE2 &
```

Comment:
Could you adjust the math of SE2 to take into consideration that the email isn't a duplicate. Something like "and email does not exist in db" in math terms if possible